src/FormNetworks/quadraticformnetwork.jl

```diff
@@ -0,0 +1,86 @@
+default_index_map = prime
+
+struct QuadraticFormNetwork{V,FormNetwork<:BilinearFormNetwork{V},IndexMap} <:
+       AbstractFormNetwork{V}
+  formnetwork::FormNetwork
+  dual_index_map::IndexMap
```

Comment:
I think there should also be an inverse map so you can map back and forth between bra and ket indices. Maybe we can call them `index_map` and `inv_index_map`?